peerdid/peer_did.py

```diff
@@ -119,21 +125,30 @@ def create_peer_did_numalgo_2(
     return peer_did
 
 
-def resolve_peer_did(peer_did: PEER_DID, version_id=None) -> JSON:
+def resolve_peer_did(
+    peer_did: PEER_DID,
+    version_id=None,
+    format: VerificationMaterialFormat = VerificationMaterialFormat.BASE58,
```

Comment:
I think `publicKeyBase58` is outdated. Can you change default value, please?

Reply:
I agree that it would be great to change the default, but at the same time I don't want to have a default different from the examples in the main spec (both peer did and didcomm specs still use 2018/2019 methods and publicKeyBase58).

So, I believe the right solution should be the following:
- Use 2020 methods and `publicKeyMultibase` as a default
- Send a PR to peer did spec changing the examples there to use 2020 methods and `publicKeyMultibase`
- Send a PR to DIDComm spec changing the examples there to use 2020 methods and `publicKeyMultibase`

I will send that PRs.